src/main/java/org/openmrs/maven/plugins/utility/ConfigurationManager.java

```diff
@@ -0,0 +1,149 @@
+package org.openmrs.maven.plugins.utility;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.log4j.Logger;
+import org.apache.maven.model.*;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.apache.maven.model.io.xpp3.MavenXpp3Writer;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.openmrs.maven.plugins.model.Artifact;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Class for reading/writing pom.xml
+ */
+public class ConfigurationManager {
+    private String path;
+    private Model model;
+
+    private static final Logger log = Logger.getLogger(ConfigurationManager.class);
+
+    /**
+     * Default constructor
+     */
+    public ConfigurationManager() {
+        model = new Model();
+    }
+
+    /**
+     * Create new configuration object by the path
+     * @param projectPath - path to project
+     */
+    public ConfigurationManager(String projectPath) {
+        this();
+        path = new File(projectPath, SDKConstants.OPENMRS_SERVER_POM).getPath();
+        File conf = new File(path);
+        FileReader reader = null;
+        if (conf.exists()) {
+            try {
+                reader = new FileReader(path);
+                model = new MavenXpp3Reader().read(reader);
```

Comment:
If you open up javadoc for closeQuitely you will notice they suggest a certain pattern to use it. It includes calling reader.close() in the try block in addition to calling closeQuietly in the finally block. That is to make sure that if an exception is thrown while closing the reader, it is also caught. If you only have closeQuitely in the finally block, the exception will be quitely ignored, which is a potentionally dangerous situation.